app/Jobs/SendPostToPhoenix.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Photo;
+use Rogue\Services\Phoenix;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendPostToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue;
+
+    /*
+     * Photo instance.
+     *
+     * @var Rogue\Models\Photo
+     */
+    protected $photo;
+
+    /*
+     * @var bool
+     */
+    protected $hasFile;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Photo $photo, $hasFile = true)
```

Comment:
Do we always know it's a `$photo`? Shouldn't this just be `$post` since it _could_ be a variety post types? Or is this because it's a specific class for Drupal Phoenix?